recipes/pytest-icdiff/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-icdiff" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f874e668d3c8e590e49a33748ebfd0449cc5537a4919e87f5c3260ac1713504
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - icdiff
+    - pprintpp
+    - pytest
+    - python >=3.6
+    - setuptools
```

Comment:
Is setuptools necessary as run requirements?

Reply:
Oops, thanks for catching that. Fixed.